persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -56,23 +59,25 @@ object HmdaFileParser {
 class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
 
   import HmdaFileParser._
+  val duration = 10.seconds
+  implicit val timeout = Timeout(duration)
+  val config = ConfigFactory.load()
+  val flowParallelism = config.getInt("hmda.actor-flow-parallelism")
 
   var state = HmdaFileParseState()
   var encounteredParsingErrors: Boolean = false
-  val submissionLarStats = context.actorSelection(s"submission-lar-stats-${submissionId.toString}")
+  val supervisor = system.actorSelection("/user/supervisor")
```

Comment:
`HmdaFileParser` is a child of `HmdaSupervisor`. You can get the supervisor by doing `context.parent`, which gives you an `ActorRef`